### PR TITLE
Add inline backticks for localstorage clear

### PR DESCRIPTION
**Pull request by alayek**
A small modification on https://github.com/FreeCodeCamp/wiki/issues/748

### CODE REVIEW

**Comment by alayek** [comment]
dont's merge, I need to update a few other stuff as @abhisekp mentioned in Wiki room.

**Comment by alayek** [comment]
Ok updated.

**Comment by Rafase282** [comment]
Seems fine to me, let me know and I'll squash and merge.

File: Clear-Your-Browser's-Local-Storage.md [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
 
 In Chrome:
-* On 'freecodecamp.com' open your console
- * Windows: Ctrl + Shift + J
- * Mac OS: Cmd + Opt + J
+* On [freecodecamp](https://freecodecamp.com) open your console
+ * Windows: `Ctrl` + `Shift` + `J`
+ * Mac OS: `Cmd` + `Opt` + `J`
 * Go to resources tab(chrome). 
  * There click on the localStorage link in the nav bar on the right.
```
Comment (abhisekp) [inline Clear-Your-Browser's-Local-Storage.md]:
```diff
- ...on the localStorage link ....
+ ...on the "Local Storage" link ....
```

**Pull request merged**